pkg/async/notifications/factory.go

```diff
@@ -95,7 +95,11 @@ func NewNotificationsPublisher(config runtimeInterfaces.NotificationsConfig, sco
 		snsConfig := gizmoAWS.SNSConfig{
 			Topic: config.NotificationsPublisherConfig.TopicName,
 		}
-		snsConfig.Region = config.Region
+		if config.AWSConfig.Region != "" {
+			snsConfig.Region = config.AWSConfig.Region
+		} else {
+			snsConfig.Region = config.Region
```

Comment:
aah this is for backwards compatibility? Can we put a deprecated on this flag? so that we can remove it eventually

Reply:
I put a deprecated comment on https://github.com/honnix/flyteadmin/pull/1/files#diff-2cd697d44b48c0070c8e0756ed47b382R141, according to https://blog.golang.org/godoc. Is there a better way for deprecation in go?